server/user-groups.ts

```diff
@@ -344,3 +344,55 @@ export class GlobalAuth extends Auth {
 		return true;
 	}
 }
+
+export class SectionLeaders extends Set<ID> {
```

Comment:
See dms. imo, this is better suited `Map<userid, section the user leads>` that's attached to Users.globalAuth. (cleaner, and keeps everything in the same place)

Reply:
Sharing from discord DMs:
> mia — Today at 12:46 AM
> ehh
> i feel like, since section leader is partially global
> it's better suited to being near the rest of the auth tables
> imo
> but that's just me
> plus, in that case like
> we wouldn't need a user.sectionLeader
> we could just like
```ts
Users.globalAuth.isSectionLeader = function(name: string) {
return this.sectionLeaders.has(toID(name)):
}```

(or, as was mentioned, just attach the SectionLeaders class to global auth)